data/json/furniture.json

```diff
@@ -1576,7 +1576,7 @@
     "type": "furniture",
     "id": "f_crate_o",
     "name": "open crate",
-    "description": "What's inside?  Look in it!",
+    "description": "What's in the box?  Look in it!",
```

Comment:
This is one of those lateral changes just for the sake of change.

Reply:
understood